runtime/queries/scala/highlights.scm

```diff
@@ -112,6 +112,9 @@
 (generic_function
   function: (identifier) @function)
 
+(interpolated_string_expression
+  interpolator: (identifier) @function.call)
```

Comment:
```suggestion
  interpolator: (identifier) @function)
```

We use different captures: https://docs.helix-editor.com/master/themes.html#syntax-highlighting

Reply:
Fixed f43f3b494d54031c44cee0165675c38ad7b721c7